rholang/README.md

```diff
@@ -4,18 +4,27 @@ Rholang is a behaviorally typed, concurrent programming language, with a focus o
 
 The language is still in the early stages of development, but for those who are interested, more information can be found in the [RChain Platform Architecture](http://rchain-architecture.readthedocs.io/en/latest/).
 
-# Configuration
+## Building from Source
 
 1. Clone the repository
 2. Configure/fetch dependencies
     * [sbt](http://www.scala-sbt.org/0.13/docs/Installing-sbt-on-Linux.html)
-    * [CUP](http://www2.cs.tum.edu/projects/cup/install.php) - can be installed using apt
-    * JLex - install using apt
-    * BNFC
-    * Scala
-4. Run `sbt bnfc:generate` to generate the parser
-5. Run `sbt console` to launch the sbt console
-6. In the sbt console import the compiler with `import coop.rchain.rho2rose._`
-7. And then compile any Rholang ".rho" file with `Rholang2RosetteCompiler.main(Array("<path_to_Rholang_file>.rho"))`
+    * [BNFC](http://bnfc.digitalgrammars.com/) v2.8.1 Nov 2016 - e.g. `cabal install bnfc`
+3. Run `sbt bnfc:generate` to generate the parser
+4. Run `sbt compile` to compile classes
+5. Run `sbt assembly` to build a stand-alone .jar file
+
+## Command-line usage
+
+    $ java -jar target/scala-2.11/rholang-assembly-0.1-SNAPSHOT.jar examples/token.rho 
+    compiled examples/token.rho to examples/token.rbl
```

Comment:
Can you wrap this in a shell script that passes its arguments to java?

Reply:
I think so.